packages/cli/src/server/api/trace.ts

```diff
@@ -0,0 +1,249 @@
+          startTime: log.timestamp,
+          endTime: log.timestamp + (log.body.duration || 0),
+          attributes: log.body.attributes || {},
+          events: log.body.events || [],
+          status: log.body.status || { code: 0 },
+          parentSpanId: log.body.parentSpanId,
+        }))
+        .sort((a, b) => a.startTime - b.startTime);
+
+      logger.info('Trace data retrieved successfully', {
+        traceId,
+        method: 'GET',
+        path: req.path,
+        spanCount: spans.length,
+      });
+
+      // Set Content-Type header and return formatted response
+      res.setHeader('Content-Type', 'application/json');
+      return res.status(200).json({
+        resourceSpans: [
+          {
+            resource: {
+              attributes: {
+                'service.name': 'eliza',
+                'service.version': process.env.npm_package_version || 'unknown',
+              },
+            },
+            scopeSpans: [
+              {
+                spans: spans.map((span) => ({
+                  traceId: span.traceId,
+                  spanId: span.spanId,
+                  parentSpanId: span.parentSpanId,
+                  name: span.name,
+                  kind: 1, // INTERNAL
+                  startTimeUnixNano: span.startTime * 1000000, // Convert to nanoseconds
+                  endTimeUnixNano: span.endTime * 1000000, // Convert to nanoseconds
+                  attributes: Object.entries(span.attributes).map(([key, value]) => ({
+                    key,
+                    value: {
+                      stringValue: String(value),
+                    },
+                  })),
+                  events: span.events.map((event) => ({
+                    timeUnixNano: event.timestamp * 1000000, // Convert to nanoseconds
+                    name: event.name,
+                    attributes: Object.entries(event.attributes).map(([key, value]) => ({
+                      key,
+                      value: {
+                        stringValue: String(value),
+                      },
+                    })),
+                  })),
+                  status: {
+                    code: span.status.code,
+                    message: span.status.message,
+                  },
+                })),
+              },
+            ],
+          },
+        ],
+      });
+    } catch (error) {
+      logger.error('Error retrieving trace data', {
+        traceId,
+        method: 'GET',
+        path: req.path,
+        error: error instanceof Error ? error.message : String(error),
+        stack: error instanceof Error ? error.stack : undefined,
+      });
+
+      return res.status(500).json({
+        error: 'Internal server error while retrieving trace data',
+      });
+    }
+  });
+
+  return router;
+};
+
+// Create the default router instance with the default database adapter
+const defaultDb = createDatabaseAdapter({}, '123e4567-e89b-12d3-a456-426614174002' as UUID);
```

Comment:
What is this UUID represent we should store it as a variable and clearly define what it is for readability and maintainability

Reply:
Also shouldn't this be part ofsome init function instead of in the root of the file